models/db/search.go

```diff
@@ -28,3 +34,14 @@ const (
 	SearchOrderByForks                 SearchOrderBy = "num_forks ASC"
 	SearchOrderByForksReverse          SearchOrderBy = "num_forks DESC"
 )
+
+func FindObjects[Object any](ctx context.Context, cond builder.Cond, opts *ListOptions, objects *[]*Object) error {
```

Comment:
What's the purpose of generics here?

Reply:
I guess to restrict the list to one specific unknown type without knowing this type in advance?

Reply:
It's a common function to find objects from database with a `cond` and `opts` which could cover many situations.